server/routes/auth.js

```diff
@@ -87,6 +88,29 @@ function validationMiddleware(req, res, next) {
     next();
 }
 
+router.get("/user", loginRequired, async function (req, res, next) {
```

Comment:
We do still want to call this on the initial page load - similar to the withCookie you had before. A function call from useEffect on a top level component that will call something like checkLogin() in userContext that makes a GET request to this route when someone visits the page so if they logged in previously they don't have to do so again

Reply:
I've added this back in.  For some reason there is a slight flicker between login and home when there is a token and I am not sure how to fix that.